File: Sorting Algorithms/Bubble_Sort.cpp
#include<iostream>
using namespace std;

int main() {
    int n;
    cout<<"Enter the number of elements: ";
    cin>>n;

    bool change = false;

    int arr[n];
    cout<<"Enter the elements: ";
    for(int i=0; i<n; i++) {
        cin>>arr[i];
    }

    for(int i=0;i<n;i++) {
        change = false;

        for(int j=0;j<n-i-1;j++) {
            if(arr[j] > arr[j+1]) {
                swap(arr[j], arr[j+1]);
                change = true;
            }
        }

        if(change == false) {
            break;
        }
    }

    cout<<"Sorted Array: ";
    for(int i=0;i<n;i++) {
        cout<<arr[i]<<" ";
    }
    
    return 0;
}

File: Sorting Algorithms/Bubble_Sort.py
n = int(input("Enter the number of elements in the list: "))
arr = []
for i in range (n):
    arr.append(int(input("Enter the element: ")))
    
for i in range(n):
    for j in range(n-i-1):
        if(arr[j] > arr[j+1]):
            arr[j], arr[j+1] = arr[j+1], arr[j]
    
print("The sorted list is: ")
for i in range(n):
    print(arr[i], end = " ")